docs/chapters/05_going-deeper.md

```diff
@@ -166,6 +166,92 @@ source <path-to-your-bash-file> && boost deploy -e production
 
 Now just let the magic happen, Booster will create everything for you and give you back your app ready to use URL. 🚀
 
+#### Azure synth command
+
+Azure provider implement the experimental **Booster** `synth` command. This command will generate [Terrafom](https://www.terraform.io/) templates from your code. It will also generate needed files to deploy your Booster application using [cdktf](https://learn.hashicorp.com/tutorials/terraform/cdktf).
```

Comment:
Question: What would happen if I call `boost synth` on the AWS provider?

Reply:
You will get this error: `Error: Error: Attempted to perform the 'synth' operation with a provider that does not support this feature, please check your environment configuration.`